maliput_ros_interfaces/msg/LaneId.msg

```diff
@@ -0,0 +1,2 @@
+# Represents a maliput::api::LaneId.
+string id
```

Comment:
In general if we only were to have Ids won't make sense to have a custom message with a sole string field. But given we expect to add several messages then makes sense to differentiate a String msg from a LaneId (or other msgs).

Reply:
Yes, I did that on purpose to cast them further to each entity type.